2022-01-18 15:43:17,366 INFO    MainThread:3956 [wandb_setup.py:_flush():71] setting env: {}
2022-01-18 15:43:17,366 INFO    MainThread:3956 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-18 15:43:17,366 INFO    MainThread:3956 [wandb_setup.py:_flush():71] setting login settings: {}
2022-01-18 15:43:17,367 INFO    MainThread:3956 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\acb6595\Documents\Research\places2\project_portable\code\src\d01_pre_processing\wandb\run-20220118_154317-33bluxko\logs\debug.log
2022-01-18 15:43:17,367 INFO    MainThread:3956 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\acb6595\Documents\Research\places2\project_portable\code\src\d01_pre_processing\wandb\run-20220118_154317-33bluxko\logs\debug-internal.log
2022-01-18 15:43:17,367 INFO    MainThread:3956 [wandb_init.py:init():404] calling init triggers
2022-01-18 15:43:17,367 INFO    MainThread:3956 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-18 15:43:17,367 INFO    MainThread:3956 [wandb_init.py:init():449] starting backend
2022-01-18 15:43:17,367 INFO    MainThread:3956 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=spawn, using: spawn
2022-01-18 15:43:17,373 INFO    MainThread:3956 [backend.py:ensure_launched():199] starting backend process...
2022-01-18 15:43:17,701 INFO    MainThread:3956 [backend.py:ensure_launched():204] started backend process with pid: 14596
2022-01-18 15:43:17,701 INFO    MainThread:3956 [wandb_init.py:init():458] backend started and connected
2022-01-18 15:43:17,702 INFO    MainThread:3956 [wandb_init.py:init():519] updated telemetry
2022-01-18 15:43:17,706 INFO    MainThread:3956 [wandb_init.py:init():549] communicating current version
2022-01-18 15:43:19,138 INFO    MainThread:3956 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.9 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-01-18 15:43:19,138 INFO    MainThread:3956 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-01-18 15:43:19,368 INFO    MainThread:3956 [wandb_init.py:init():592] starting run threads in backend
2022-01-18 15:43:20,342 INFO    MainThread:3956 [wandb_run.py:_console_start():1816] atexit reg
2022-01-18 15:43:20,343 INFO    MainThread:3956 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.WRAP
2022-01-18 15:43:20,343 INFO    MainThread:3956 [wandb_run.py:_redirect():1727] Wrapping output streams.
2022-01-18 15:43:20,344 INFO    MainThread:3956 [wandb_run.py:_redirect():1751] Redirects installed.
2022-01-18 15:43:20,344 INFO    MainThread:3956 [wandb_init.py:init():619] run started, returning control to user process
2022-01-18 15:46:11,466 INFO    MainThread:3956 [wandb_run.py:finish():1463] finishing run austinbergstrom/places_dry_run/33bluxko
2022-01-18 15:46:11,466 INFO    MainThread:3956 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2022-01-18 15:46:11,467 INFO    MainThread:3956 [wandb_run.py:_restore():1758] restore
2022-01-18 15:46:12,132 INFO    MainThread:3956 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 655
  total_bytes: 655
}

2022-01-18 15:46:12,712 INFO    MainThread:3956 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 655
  total_bytes: 655
}

2022-01-18 15:46:12,912 INFO    MainThread:3956 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 655
  total_bytes: 655
}

2022-01-18 15:46:13,509 INFO    MainThread:3956 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 655
  total_bytes: 4338
}

2022-01-18 15:46:13,612 INFO    MainThread:3956 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 655
  total_bytes: 4338
}

2022-01-18 15:46:13,715 INFO    MainThread:3956 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 655
  total_bytes: 4338
}

2022-01-18 15:46:13,817 INFO    MainThread:3956 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 655
  total_bytes: 4338
}

2022-01-18 15:46:13,918 INFO    MainThread:3956 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4338
  total_bytes: 4338
}

2022-01-18 15:46:14,021 INFO    MainThread:3956 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4338
  total_bytes: 4338
}

2022-01-18 15:46:14,122 INFO    MainThread:3956 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4338
  total_bytes: 4338
}

2022-01-18 15:46:14,226 INFO    MainThread:3956 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4338
  total_bytes: 4338
}

2022-01-18 15:46:14,528 INFO    MainThread:3956 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4338
  total_bytes: 4338
}

2022-01-18 15:46:15,003 INFO    MainThread:3956 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4338
  total_bytes: 4338
}
local_info {
}

2022-01-18 15:46:16,148 INFO    MainThread:3956 [wandb_run.py:_append_files():2184] logging synced files
